run.py

```diff
@@ -74,13 +77,17 @@ def validate_config(fields, section):
 
 def list_workspaces(username, password, iam_apikey, version, url):
     authenticator = IAMAuthenticator(iam_apikey)
-    c = AssistantV1(
-        version=version,
-        authenticator=authenticator
-    )
-    c.set_service_url(url)
-
-    return c.list_workspaces()
+    if WATSON_SERVICE == 'assistant':
```

Comment:
Devi, can you alter the if tests around "not NLC" rather than "is assistant" - this provides additional safety to the default case as well as making it slightly easier if even other modes are added.